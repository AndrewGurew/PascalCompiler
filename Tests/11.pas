a = array[1001..12123] of integer;