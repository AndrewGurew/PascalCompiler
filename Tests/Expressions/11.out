Unknown identifier a
Unknown identifier b
Unknown identifier c
⎬-
 ⎬a
 ⎬*
  ⎬/
  ⎪⎬b
  ⎪⎬c
  ⎬f